{"uuid": "a369af4a-9117-4766-a8e8-71ebcac9d909", "children": ["8417d2a5-4bc9-4b0a-9f36-3ec82d778118"], "befores": [{"name": "api_client", "status": "broken", "statusDetails": {"message": "AttributeError: 'ConfigProvider' object has no attribute 'config'\n", "trace": "  File \"C:\\Users\\user\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\user\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\user\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\user\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\user\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\fixtures.py\", line 898, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"C:\\Users\\user\\Desktop\\skypro\\final_proj\\pytest_ui_api_template\\test\\conftest.py\", line 23, in api_client\n    url = ConfigProvider().get(\"api\", \"base_url\")\n  File \"C:\\Users\\user\\Desktop\\skypro\\final_proj\\pytest_ui_api_template\\conficuration\\configprovider.py\", line 11, in get\n    return self.config[section].get(prop)\n           ^^^^^^^^^^^\n"}, "start": 1737711105592, "stop": 1737711105592}], "start": 1737711105592, "stop": 1737711105602}